src/core/LicenseService.ts

```
@@ -0,0 +1,33 @@
+    return this.replacePlaceholders(data.body, { fullname, year });
+  }
+
+  static replacePlaceholders(
+    text: string,
+    { fullname, year }: { fullname: string; year?: number },
+  ): string {
+    return text
+      .replace(/\[fullname\]/g, fullname)
+      .replace(/\[year\]/g, String(year || new Date().getUTCFullYear()));
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)